lib/plugins/hmr.js

```diff
@@ -20,8 +20,8 @@ const addPlugin = (compiler) => {
 const init = function init(compiler, log) {
   // eslint-disable-next-line no-param-reassign
   compiler.options.output = Object.assign(compiler.options.output, {
-    hotUpdateChunkFilename: `${compiler.wpsId}-[id]-wps-hmr.js`,
-    hotUpdateMainFilename: `${compiler.wpsId}-wps-hmr.json`
+    hotUpdateChunkFilename: `[runtime]-${compiler.wpsId}-[id]-wps-hmr.js`,
```

Comment:
Is `[runtime]` available in webpack v4 as well, or will this just prefix the literal `"[runtime]"` onto the files using webpack v4?

Reply:
Correct, it will be only prefix [runtime]-***.js

Reply:
OK let's only apply the prefix if we're in webpack v5 then. I believe there are some checks for the version in a few other places in the repo that you can use as an example.

Reply:
I was wrong, it was in a different repo: https://github.com/shellscape/webpack-nano/pull/22/files#diff-544803892ac752aa674ffa81380c022af562eda776860d94eb5c2d3bdb59c99cR12-R14

Reply:
Thank you! I added the helpers file because I couldn't find the correct place to save the new function - getMajorVersion and 2 tests added.

Reply:
Ups, sorry, I didn't see your the last message (( Will fix

Reply:
I faced the same CI error. It works well on my local PC and when I made a change up timeout 30 000 it worked well too but now it's an error (((

Reply:
yeah, I think we need to rework those integration tests. they've started to fail on CI on a regular basis now.